mkposters/mkposter.py

```diff
@@ -21,25 +21,30 @@ def md_to_html(md: str):
     )
 
 
+def excludeHiddenDirs(path, subdirs):
```

Comment:
I'm guessing you're coming from another language. In Python the convention is that functions use `snake_case` rather than `lowerCamelCase`.